design-patterns/domain-driven-design/pattern1/README.md

```diff
@@ -0,0 +1,193 @@
+# Pattern 1: Single Domain, Single Bounded Context, and a Single Uni
+
+
+## Scenario
+
+The Customer Relationship Management(CRM) Team wants to create a single source of truth for customer account data, which includes firstname, lastname, customerId, personal details, contact details, and the address. The CRM Team will frequently query customer data using customerId. 
+
+
+## Modeling
+
+Since CRM is an independent business unit we will assume it is a domain on its own. Inside CRM, we will have multiple sub-domains; customer account is one such sub-domain. You will model customer account as a Bounded Context.  Since CRM will perform frequent lookup, search, and filter on customer records by customerId, you will index customer accounts on the customerId property. 
+
+If you are confused between Bounded Context, Entity, and the Aggregate. These terms correlate 1:1 for the purpose of this exercise. I will use the term Entity here to mean Root Entity which is an Aggregate in DDD. And, Bounded Context can have one or more than one Entity.
+
+
+
+
+## Create a Uni
+
+1. Login to [Vendia Share](https://share.vendia.net/login). 
+
+Don’t have a login, you can [sign up](https://share.vendia.net/) now. 
+
+2. Create a Uni
+
+Click the `Create Universal Application` button on the top-right. Next, use the `Create your own` option. 
+
+3. Fill out details in the Uni creation wizard. 
+
+**Step 1:** Give your Uni a name. 
+Prefix your Uni name with prefix `test-` so its name can be reused. Pick a unique Uni name since all Starter and Individual Tier users will share the vendia.net namespace, which will cause naming collisions.
+Since this Uni will be owned and operated by the CRM Team. I will give it the name `test-CRM-append-unique-suffix`. Time to get creative, replace `append-unique-suffix` with your unique suffix in the Uni name. In this blog, I will simply refer to new Uni as the `CRM`. Click `Next`. 
+
+**Step 2:** Fill out the following information to create a Uni node. 
+
+```
+Node Name: "CRM". // you can change the name.
+Node Description: "Node dedicated to the CRM Domain". // You can change the description.
+Cloud Server Provider: "AWS".
+Node Region:"us-east-1". //you can change it to other regions.
+Auth Option: "Vendia User". // Let's use Vendia User
```

Comment:
Will the GraphQL Explorer work if Auth is set to `Vendia User`?

Separately, I really think it's a better idea to use the CLI to deploy the Uni. The callouts to sections in the schema.json file is a great idea, but I think having a more prescriptive one-liner will lead to a better experience. We've used the convention of having a directory `uni_configuration` and storing a schema.json, registration.json.sample, and (optional) init-state.json. Refer to the examples in https://github.com/vendia/examples/tree/main/share.

Reply:
GraphQL Explorer works fine.  I thought about CLI. But, I decided to keep it simple for users. This post is more about demonstrating an architecture and less about how a developer or a DevOps person should actually use Vendia Share in an organization.